test/ILLink.RoslynAnalyzer.Tests/LinkerTestCases.cs

```diff
@@ -22,9 +22,12 @@ public void RequiresCapability (MethodDeclarationSyntax m, List<AttributeSyntax>
 			case "TestCovariantReturnCallOnDerived":
 			case "TestRequiresUnreferencedCodeOnlyThroughReflection":
 			case "TestStaticCctorRequiresUnreferencedCode":
+			case "TestStaticCtorMarkingIsTriggeredByFieldAccess":
 			case "TestTypeWhichOverridesMethodVirtualMethodRequiresUnreferencedCode":
 			case "TestRequiresInMethodFromCopiedAssembly":
 			case "TestRequiresThroughReflectionInMethodFromCopiedAssembly":
+			case "TestStaticCctorTriggeredByMethodCall":
```

Comment:
Extra c in the name "TestStaticCctorTriggeredByMethodCall" vs "TestStaticCtorTriggeredByMethodCall"